.gitignore

```diff
@@ -18,7 +18,18 @@ src/tool/users_balance.csv
 src/tool/record_transactions_*.csv
 
 *.DS_Store
-configs
+charts/configs/*
+!charts/configs/lnd
+charts/configs/lnd/*
+!charts/configs/lnd/regtest.yaml
```

Comment:
those files are not commited in this PR?

Reply:
They are